deeprank2/utils/buildgraph.py

```diff
@@ -209,16 +209,16 @@ def get_residue_contact_pairs( # pylint: disable=too-many-locals
     structure: PDBStructure,
     chain_id1: str,
     chain_id2: str,
-    distance_cutoff: float,
+    interaction_radius: float,
 ) -> List[Pair]:
-    """Get the residues that contact each other at a protein-protein interface.
+    """Find all residue pairs that may influence each other.
 
     Args:
         pdb_path (str): The path of the pdb file, that the structure was built from.
         structure (:class:`PDBStructure`): From which to take the residues.
         chain_id1 (str): First protein chain identifier.
         chain_id2 (str): Second protein chain identifier.
-        distance_cutoff (float): Max distance between two interacting residues.
+        interaction_radius (float): Maximum distance between residues to consider them as interacting.
```

Comment:
since edges represent interactions, then isn't `interaction_radius` conceptually the same thing as `max_edge_distance`? Maybe the radius doesn't have to be necessarily about interactions (thus could be `radius` only), while edges should always be about interactions. Otherwise, what's the conceptual difference between these two variables?

Reply:
Edges don't necessarily equate to interactions. I think mostly this will be the case, and by default I would use the same value (and we do), but there could be situations where `interaction_radius` is higher than `max_edge_distance`. This could be the case 1) purely to decrease computational power/time, or 2) e.g. because the "secondary" interactions can be relevant. Even if two nodes don't have an edge between them, an intermediate node could be affected by both edges.
I can't off the top of my head think of a situation where `max_edge_distance` would be larger than `interaction_radius`, and am not sure that would even make sense to do. Maybe we can create a waning/error/cap in case the user sets cutoff > radius. What do you think?

Note that in `interaction_radius` I am not necessarily talking about protein-protein-interactions (trans interactions), but could also be cis-interactions (so residues of the same protein interacting with each other). So also in a variant, if residues are too far away, they would not interact with/affect the variant residue so don't need to be considered. 

I agree that the terminology is confusing, though. At the same time, I find `radius` too generic, it doesn't really describe what the parameter is either. Let's see if we can come up with another nomenclature. Maybe something along the lines of `sphere_of_influence`, although that sounds too fuzzy to me as well.

Reply:
as above, "radius" also has a specific meaning in graph theory.

Reply:
influence_radius